stock_picking_invoice_product_group/models/stock_move.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, api
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _get_partner_id(self):
```

Comment:
Why are you creating these methods? With mapped function you can retrieve them very easily